jsPrayer.html

```diff
@@ -20,5 +22,27 @@
 
         </script>
 
+        <!-- How to curse someone
```

Comment:
Wow, how to curse someone?

Reply:
I wouldn't touch Haskell with a 10 foot pole